ui/rogue/presets.ts

```diff
@@ -1,138 +1,141 @@
-import { BattleElixir } from '../core/proto/common.js';
+import { BattleElixir, Flask } from '../core/proto/common.js';
 import { Conjured } from '../core/proto/common.js';
 import { Consumes } from '../core/proto/common.js';
 
 import { EquipmentSpec } from '../core/proto/common.js';
 import { Food } from '../core/proto/common.js';
 import { Potions } from '../core/proto/common.js';
-import { WeaponImbue } from '../core/proto/common.js';
 import { SavedTalents } from '../core/proto/ui.js';
 
 import {
 	Rogue_Rotation as RogueRotation,
 	Rogue_Rotation_Builder as Builder,
 	Rogue_Options as RogueOptions,
+  Rogue_Options_PoisonImbue as Poison,
 } from '../core/proto/rogue.js';
 
 import * as Tooltips from '../core/constants/tooltips.js';
 
-// Preset options for this spec.
-// Eventually we will import these values for the raid sim too, so its good to
-// keep them in a separate file.
-
 // Default talents. Uses the wowhead calculator format, make the talents on
 // https://wowhead.com/wotlk/talent-calc and copy the numbers in the url.
+// https://www.wowhead.com/wotlk/talent-calc/rogue/_001xrm11xrr21qf1
 export const CombatTalents = {
 	name: 'Combat',
 	data: SavedTalents.create({
-		talentsString: '00532000532-0252051000035015223100501251',
+		talentsString: '00532000523-0252051050035010223100501251'
 	}),
 };
 
 export const DefaultRotation = RogueRotation.create({
-	builder: Builder.Auto,
+	builder: Builder.SinisterStrike,
 	maintainExposeArmor: false,
 	useRupture: true,
-	useShiv: true,
+	useShiv: false,
+	useEnvenom: false,
 	minComboPointsForDamageFinisher: 3,
 });
 
 export const DefaultOptions = RogueOptions.create({
+  mhImbue: Poison.DeadlyPoison,
+  ohImbue: Poison.InstantPoison,
 });
 
 export const DefaultConsumes = Consumes.create({
 	defaultPotion: Potions.PotionOfSpeed,
 	defaultConjured: Conjured.ConjuredRogueThistleTea,
-	battleElixir: BattleElixir.ElixirOfDeadlyStrikes,
-	food: Food.FoodFishFeast,
-	mainHandImbue: WeaponImbue.WeaponImbueRogueInstantPoison,
-	offHandImbue: WeaponImbue.WeaponImbueRogueDeadlyPoison,
+	startingConjured: Conjured.ConjuredRogueThistleTea,
```

Comment:
If this is the same as defaultConjured, then both this and `numStartingConjured` can be removed.

Reply:
Fixed. I don't completely understand how this is used but with only the default specified, the UI option for starting conjured is set to none. So, a nice side effect of having all 3 was that the value was populated (to thistle tea in my case)